index.html

```diff
@@ -74,6 +75,220 @@ <h1 class="main-text">I’m Giovanni Glad to see you!</h1>
+                    <div>
+                        <ul class="connect-icons">
+                            <li>
+                                <a href="">
+                                    <img class="twitter" src="./Assets/twitter.svg" alt="Connect Twitter">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="Linkedin" src="./Assets/Linkedin icon.svg" alt="Connect Linkedin">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="medium" src="./Assets/medium.svg" alt="Connect Medium">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="github" src="./Assets/github.svg" alt="Connect Github">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="angellist" src="./Assets/angellist.svg" alt="Connect Angel List">
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="button-resume-div">
+                    <button class="button-resume">Get my Resume</button>
+                </div>
+
+                <div class="languages">
+                    <div class="language-title">
+                        <p>Languages</p>
+                    </div>
+                    <div class="languages-list">
+                        <ul>
+                            <li class="languages-li">
+                                <img src="/Assets/javascript_circle.png" alt="">
+                                <p>JAVASCRIPT</p>
+                            </li>
+                            <li class="languages-li">
+                                <img src="/Assets/html_circle.png" alt="">
+                                <p>HTML</p>
+                            </li>
+                            <li class="languages-li">
+                                <img src="/Assets/css_circle.png" alt="">
+                                <p>CSS</p>
+                            </li>
+                        </ul>
+                    </div>
+
+                    <div class="framework-title">
+                        <p>Frameworks</p>
+                        <img src="./Assets/Union.png" alt="">
+                    </div>
+                    <div class="framework-title">
+                        <p>Skills</p>
+                        <img src="./Assets/Union.png" alt="">
+                    </div>
+                </div>
```

Comment:
- [ ] Instead of using `div` kindly use `ul` and `li` tags to gather all three things (languages, frameworks, skills) due to SEO purpose 👍

Like Below ⬇️

```
<ul>
  <li>Languages</li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```